src/util/datablock/datablock.h

```diff
@@ -12,13 +12,30 @@
 #include "../block.h"
 #include "./datablock_iterator.h"
 
-#define DELETED_MARKER 0xFF
-
 typedef void (*fpDestructor)(void *);
 
 // Number of items in a block. Should always be a power of 2.
 #define DATABLOCK_BLOCK_CAP 16384
 
+// Returns the item header size.
+#define ITEM_HEADER_SIZE (sizeof(DataBlockItemHeader))
```

Comment:
Are we expecting `sizeof(DataBlockItemHeader)` to return 1?
this is what we want, but I'm not sure this is the case,
I think it would be better to switch to:
`#define ITEM_HEADER_SIZE 1`

Reply:
changed